.travis.yml

```diff
@@ -0,0 +1,25 @@
+jdk:
+  - oraclejdk8
+
+language: scala
+
+scala:
+  - 2.12.2
+
+sudo: required
+
+#services:
+#  - docker
+
+#before_install:
+#  - docker rm -f mysql || true          # in case the after_success step does not run
+#  - docker pull mysql/mysql-server:5.7
+#  # port 3306 is not available because Travis runs its own MySQL after 2006-12-01
+#  - docker run -p 3307:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
+
+script:
+  - SBT_OPTS=-Xmx3g sbt clean coverage test coverageReport -Denv.type=test && SBT_OPTS=-Xmx3g sbt coverageAggregate #-Dmysql.host=localhost -Dmysql.port=3307 it:compile
```

Comment:
Minor, could add the system properties to `SBT_OPTS` like:
```
  - SBT_OPTS="-Xmx3g -Denv.type=test" sbt clean coverage test coverageReport && SBT_OPTS="-Xmx3g" sbt coverageAggregate
```
  
            
  Write  Preview

Minor, could add the system properties to `SBT_OPTS` like:
```
  - 
Attach files by dragging & dropping,  Choose Files selecting them, or pasting from the clipboard.
 Styling with Markdown is supported Start a review Add single comment Cancel
 +
 +after_success:
 +  - SBT_OPTS=-Xmx3g sbt coveralls
 +#  - docker rm -f mysql